.github/workflows/cla.yml

```diff
@@ -7,7 +7,9 @@ on:
 
 jobs:
   CLABot:
-    if: github.event_name == 'pull_request_target' || contains(github.event.comment.html_url, '/pull/')
+    if: |
+      (github.event_name == 'pull_request_target' || contains(github.event.comment.html_url, '/pull/')) &&
+      github.ref != 'refs/heads/l10n_crowdin_action'
```

Comment:
I just dismissed my review because I had the thought - could a third-party contributor name their branch this to skip the CLA bot check? i.e. if someone creates a branch with this name on a fork, would this check pass on their PR?

Reply:
Maybe we can also ensure the author is `github-actions` here. Or that it's not a fork. Something like that.

Reply:
yes. I believe that is one small caveat. if a third party contributor opens a PR from a branch `l10n_crowdin_action` this check will not run. we could just keep an eye out for that? alternatively there might be a better way to lock that branch down? but it's a good point. idk if there's much we can do to control branch names on forks :shrug: